src/test/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesTest/upgradeFrom_0_10/config.xml

```diff
@@ -31,6 +31,7 @@
               <name>maven</name>
               <image>maven:alpine</image>
               <privileged>false</privileged>
+              <runAsUser>10000</runAsUser>
```

Comment:
Please revert all changes to `upgrade*` files, they shouldn't be needed.

Reply:
so I have reverted all upgrade* section 
the test can you please guide for it? 
I have added it in 
```
@Test
    public void testCopyConstructorCreatesEqualInstance() {
        ContainerTemplate originalTemplate = new ContainerTemplate("myname", "myimage");
        originalTemplate.setPrivileged(true);
        originalTemplate.setRunAsUser(Integer.toUnsignedLong(10000));
        originalTemplate.setAlwaysPullImage(true);
        originalTemplate.setWorkingD
```